apps/web-client/src/routes/orders/suppliers/[id]/+page.svelte

```diff
@@ -90,28 +17,32 @@
 		lastUpdated: new Date()
 	};
 
-	let selectedBooks: string[] = [];
+	let selectedBooks: string[] = data?.books.map((book) => book.isbn) ?? [];
+
 	$: totalAmount = selectedBooks.reduce((acc, isbn) => {
-		const book = books.find((b) => b.isbn === isbn);
+		const book = data?.books[isbn];
```

Comment:
Accessing the `books`/`orderLine` array with "isbn: string" won't give us the result we need here? If we return book data in the query then we can just access `price` directly